tsconfig.json

```diff
@@ -36,8 +39,8 @@
         // "noFallthroughCasesInSwitch": true,    /* Report errors for fallthrough cases in switch statement. */
 
         /* Module Resolution Options */
-        // "moduleResolution": "node",            /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */
-        // "baseUrl": "./",                       /* Base directory to resolve non-absolute module names. */
+        "moduleResolution": "node" /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */,
+        "baseUrl": "./" /* Base directory to resolve non-absolute module names. */,
```

Comment:
I think the other changes only can resolve the problem, can't it?

Reply:
For sure. Fixed